a.go

```diff
@@ -7,3 +7,7 @@ import (
 func aFunc() {
 	fmt.Println("the following sentence is true. the preceding sentence is false.")
 }
+
+func rad() {
```

Comment:
baddd practice here...

Reply:
@CodeLingoBot capture

Reply:
@stagingcodelingobot capture this is a rule with a title